model/Dataset/Classes/Dataset.md

```diff
@@ -58,3 +58,7 @@ Metadata information that can be added to a a dataset that maybe used in a softw
   - type: xsd:string
   - minCount: 0
   - maxCount: 1
+- datasetAvailability
+  - type: xsd:string
```

Comment:
I presume it is planned to use the newly introduced vocabulary here, not just a string?

Reply:
Yes, it's to use the new definitions.  What did we not do correctly?   Nic listed all the possible 'strings' that would be accepted for this field.   @nicoschif with  the new concept of 'models pretrained with data, should we add an additional  term to the list?

Reply:
The field should then state 
`- type: datasetAvailabilityType`
(provided the vocabulary is renamed to end with "Type")